src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,46 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE_ARRAY = 10;
+    private int size = 0;
+    private final K[] keys;
+    private final V[] values;
```

Comment:
First final than non fina, also size is 0 by default
```suggestion
    private static final int MAX_SIZE_ARRAY = 10;
    private final K[] keys;
    private final V[] values;
    private int size;
```

Reply:
This comment is not resolved